hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -889,6 +898,9 @@ public Response completeMultipartUpload(@PathParam("bucket") String bucket,
 
     OmMultipartUploadCompleteInfo omMultipartUploadCompleteInfo;
     try {
+      OzoneBucket ozoneBucket = volume.getBucket(bucket);
+      S3Owner.verifyBucketOwnerCondition(headers, bucket, ozoneBucket.getOwner());
+
```

Comment:
Nit: The following `getClientProtocol().completeMultipartUpload` can be changed to use `ozoneBucket#completeMultipartUpload` instead.

This is just a preference and can be ignored. Using `getClientProtocol()` avoids the OM getBucket call, but since it's called anyway, we can use `OzoneBucket`.